dao/src/main/kotlin/repositories/infrastructureservice/InfrastructureServicesTable.kt

```diff
@@ -58,38 +57,6 @@ object InfrastructureServicesTable : SortableTable("infrastructure_services") {
 
 class InfrastructureServicesDao(id: EntityID<Long>) : LongEntity(id) {
     companion object : SortableEntityClass<InfrastructureServicesDao>(InfrastructureServicesTable) {
```

Comment:
Can you add a short description to the commit message explaining why this code became dead?